spec.html

```diff
@@ -119,11 +116,9 @@ <h1>NotifyWaiter ( _WL_, <del>_W_</del><ins>_waiterRecord_</ins> )</h1>
       a WaiterList _WL_ and <del>an agent signifier _W_</del><ins>a Record _waiterRecord_</ins>. It performs the following steps:</p>
     <emu-alg>
       1. Assert: The calling agent is in the critical section for _WL_.
-      1. Assert: <del>_W_</del><ins>_waiterRecord_</ins> is on the list of waiters.
```

Comment:
Why did you delete this assertion? (Was it always invalid, since this happens after RemoveWaiters?)

Reply:
Yeah, exactly.

Reply:
It'd be nice to make this a separate PR to ecma262 master, for clarity.